packages/plugins/eslint-plugin-react-x/src/rules/no-useless-fragment.spec.ts

```diff
@@ -178,5 +196,13 @@ ruleTester.run(RULE_NAME, rule, {
         },
       },
     },
+    {
+      code: /* tsx */ `{foo}`,
```

Comment:
I wonder if this should actually be reported as an error

Reply:
This is a regular JavaScript equivalent of:
```js
{
  foo;
}
```
Some other lint rules may be triggered in certain contexts, but not by this rule.